input/icinga2.go

```diff
@@ -0,0 +1,358 @@
+package input
+
+import (
+	"time"
+	"bytes"
+	"strings"
+	"net/http"
+	"text/template"
+	"encoding/json"
+	"github.com/dustin/go-humanize"
```

Comment:
This is a new dependency. Could you please run `dep` to also update the Lockfile